src/DynamoCoreWpf/Controls/IncanvasSearchControl.xaml

```diff
@@ -23,8 +23,8 @@
             <Style x:Key="SearchMemberStyle"
                    BasedOn="{StaticResource MemberGroupMemberStyle}"
                    TargetType="{x:Type ListBoxItem}">
-                <EventSetter Event="PreviewMouseLeftButtonDown"
-                             Handler="OnMouseLeftButtonDown" />
+                <EventSetter Event="PreviewMouseLeftButtonUp"
+                             Handler="OnMouseLeftButtonUp" />
```

Comment:
It's because of dragging scroll bar.
Member button is "under" scrollviewer. So, if we click on member button or scroll bar, actually we click on scrollviewer. That's why we check on Up event, what was the original sender. If it was button - create node. If it was Thumb - do not create node.